_docs/release-notes.md

```diff
@@ -378,6 +384,12 @@ Visit [https://tidepool.org/uploader](https://tidepool.org/uploader) to download
 
 If you already have the Tidepool Uploader installed on your computer, it will automatically update to the latest version.  
 
+### 2.7.4 (Released 2018-08-10)
+The Tidepool Uploader now automatically generates a raw data file called a Binary Blob when an error occurs during an upload from FreeStyle Libre and all supported Medtronic pumps and store them in a secure location for our engineers to access when it's time to squash some bugs. Between this and Rollbar support we added last November, turnaround times from us on feature fixes should improve tremendously.  
```

Comment:
We not only store it when an error occurs, but with all uploads. This will eventually allow us to re-process data if we change our data model in future.
The "Binary Blob" name is just something we use internally to refer to raw data, maybe we should just call it raw data? How about:

The Tidepool Uploader now saves not only processed data, but also the raw binary data retrieved from FreeStyle Libre and all supported Medtronic pumps. The data is stored in a secure location for our engineers to access when it's time to squash some bugs or if re-processing is required. Between this and Rollbar support we added last November, turnaround times from us on bug fixes should improve tremendously.

Reply:
I like your suggestion. I'll make that change and merge away. Thanks.